### DISCUSSION TITLE
Get the Resultcount before building File

### DISCUSSION THREAD

**Discussion by JavaBKBC** [firegloves/MemPOI#21] [Q&A]
Is it possible to get the resultcount before genereating the file? I have the cases, where the result of a query is empty and in thsi case, I don't want to get the Excel-Sheet.

**Comment by mirkoPan** [comment]
Hi @JavaBKBC,
it is a pleasure to see your interest in the library.
Currently this feature is not supported but it is a development that we will put in the pipeline.
For now, as a workaround, since you know the dataset you pass to the library, we advise you to manually skip the creation of the file in case the data is not present. Clearly this will involve another call to the DB from you but it should work for now ;)
